### PR TITLE
[test visibility] Update "new test" logic for early flake detection

### PR DESCRIPTION
**Pull request by juan-fernandez**
### What does this PR do? What is the motivation?

The "new test" logic has changed in our backend. We need to reflect this in the docs. I've removed the explanation image until we update it, because that's not the logic anymore. 

### Merge instructions

- [x] Please merge after reviewing

### Additional notes

https://docs-staging.datadoghq.com/juan-fernandez/update-efd-logic/tests/early_flake_detection/?tab=javascripttypescript

### CODE REVIEW

**Review by juliomalegria-dd — APPROVED** [review]

File: content/en/tests/early_flake_detection/_index.md [hunk 1]
```diff
@@ -94,11 +94,7 @@ The test framework compatibility is the same as [Test Visibility Compatibility][
 
 ## Manage Excluded Branches
 
-Excluded Branches will not have any tests retried by Early Flake Detection. Tests run in these branches are not considered new for the purposes of Early Flake Detection.
-
-{{< img src="continuous_integration/early_flake_detection_commit_new_test_explanation.png" alt="How Early Flake Detection works in your commits" style="width:100%">}}
-
-You can manage the list of excluded branches on the [Test Service Settings page][7], ensuring that the feature is tailored to your specific workflow and branch structure.
+Excluded Branches will not have any tests retried by Early Flake Detection. Tests run in these branches are not considered new for the purposes of Early Flake Detection. You can manage the list of excluded branches on the [Test Service Settings page][7], ensuring that the feature is tailored to your specific workflow and branch structure.
```
Comment (aliciascott) [inline content/en/tests/early_flake_detection/_index.md:97]:
```suggestion
Excluded Branches do not have any tests retried by Early Flake Detection. Tests run in these branches are not considered new for the purposes of Early Flake Detection. You can manage the list of excluded branches on the [Test Service Settings page][7], ensuring that the feature is tailored to your specific workflow and branch structure.
```

File: content/en/tests/early_flake_detection/_index.md [hunk 2]
```diff
@@ -115,7 +111,7 @@ If you suspect there are issues with Early Flake Detection, navigate to the [Tes
 
 This could be caused by a couple of reasons:
 
-* This test has already run in an excluded branch, such as `staging`, `main`, or `preprod`.
+* This test has already run before.
```
Comment (aliciascott) [inline content/en/tests/early_flake_detection/_index.md:114]:
```suggestion
* This test has ran previously.
```

**Review by aliciascott — CHANGES_REQUESTED** [review]
Hi @juan-fernandez just a couple small things on this one and also can you add back the image itself? It's ok to remove that `img` tag in the markdown but just physically leave the image in the repo as it will take a few days to update on the non-English docs and we don't want it to display as a broken image in the meantime, thanks!

**Comment by juan-fernandez** [comment]
> Hi @juan-fernandez just a couple small things on this one and also can you add back the image itself? It's ok to remove that `img` tag in the markdown but just physically leave the image in the repo as it will take a few days to update on the non-English docs and we don't want it to display as a broken image in the meantime, thanks!

that makes sense, thanks. I added it back

**Pull request merged**